service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/FurtherEvidenceUploadedEventHandler.java

```diff
@@ -32,34 +38,76 @@ public void handleDocumentUploadedEvent(final FurtherEvidenceUploadedEvent event
                 recipients.addAll(furtherEvidenceNotificationService.getLocalAuthoritySolicitorEmails(caseData));
             }
         }
 
         recipients.removeIf(email -> Objects.equals(email, uploader.getEmail()));
 
         if (isNotEmpty(recipients)) {
-            furtherEvidenceNotificationService.sendNotification(caseData, recipients, uploader.getFullName());
+            furtherEvidenceNotificationService.sendNotification(caseData, recipients, uploader.getFullName(),
+                newNonConfidentialDocuments);
+        }
+    }
+
+
+    private List<String> getNewNonConfidentialBundleNames(CaseData caseData, CaseData caseDataBefore,
+                                                          DocumentUploadNotificationUserType userType) {
+
+        List<Element<SupportingEvidenceBundle>> newBundle = getEvidenceBundle(caseData, userType);
+        List<Element<SupportingEvidenceBundle>> oldBundle = getEvidenceBundle(caseDataBefore, userType);
+
+        List<String> documentNames = new ArrayList<String>();
+
+        unwrapElements(newBundle).forEach(newDoc -> {
+            if (!newDoc.isConfidentialDocument() && !unwrapElements(oldBundle).contains(newDoc)) {
+                documentNames.add(newDoc.getName());
+            }
+        });
+        return documentNames;
+    }
+
+    private List<Element<SupportingEvidenceBundle>> getEvidenceBundle(CaseData caseData,
+                                                                      DocumentUploadNotificationUserType userType) {
+        if (userType == LOCAL_AUTHORITY) {
+            return caseData.getFurtherEvidenceDocumentsLA();
+        } else if (userType == SOLICITOR) {
+            List<Element<SupportingEvidenceBundle>> furtherEvidenceBundle =
+                caseData.getFurtherEvidenceDocumentsSolicitor();
+            List<Element<SupportingEvidenceBundle>> respondentStatementsBundle =
+                getEvidenceBundleFromRespondentStatements(caseData);
+
+            return concatEvidenceBundles(furtherEvidenceBundle, respondentStatementsBundle);
+        } else {
+            return caseData.getFurtherEvidenceDocuments();
         }
     }
 
-    private static boolean hasNewNonConfidentialDocuments(List<Element<SupportingEvidenceBundle>> newEvidenceBundle,
-                                                          List<Element<SupportingEvidenceBundle>> oldEvidenceBundle) {
-        List<SupportingEvidenceBundle> oldEvidenceBundleUnwrapped = unwrapElements(oldEvidenceBundle);
-        return unwrapElements(newEvidenceBundle).stream()
-            .anyMatch(d -> oldEvidenceBundleUnwrapped.stream()
-                .noneMatch(old -> old.getDocument().equals(d.getDocument()))
-                && !d.isConfidentialDocument());
+    private List<Element<SupportingEvidenceBundle>> getEvidenceBundleFromRespondentStatements(CaseData caseData) {
+        List<Element<SupportingEvidenceBundle>> evidenceBundle = new ArrayList<Element<SupportingEvidenceBundle>>();
```

Comment:
`new ArrayList<Element<SupportingEvidenceBundle>>()` should be able to simply be `new ArrayList<>()`, java will be able to infer the type arguments